content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -106,6 +115,13 @@ sudo mv minikube /usr/local/bin
 
 {{% /tab %}}
 {{% tab name="Windows" %}}
+
+You'll need to use a computer with hardware virtualization enabled. You can
+use `systeminfo` to [check](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/reference/hyper-v-requirements#verify-hardware-compatibility) whether your PC has this support.
+Although `systeminfo` mentions about Hyper-V, the status of hardware support
+for virtualization is also applicable to VirtualBox and other hypervisor
+applications.
+
```

Comment:
Hi, @sftim. Just read through again. Looks good 😄. Do you think the virtualization paragraphs should be below the `kubectl` instructions, and with the hypervisor links?
Would be great to get a `minikube` owner to review.

Reply:
>  Would be great to get a minikube owner to review.
I agree! I've moved the wording around, so I'm reasonably confident that the changed documentation remains correct, but in truth I've never installed Minikube myself.

Reply:
OK. I used this content as a refresher for Windows, but ended up consulting:
https://github.com/kubernetes/minikube#installation. Could try and ping the minikube channel on Slack?